python_modules/dagster/dagster_tests/definitions_tests/test_definitions_class.py

```diff
@@ -306,22 +311,48 @@ def one():
 
 
 def test_custom_loggers_in_definitions():
-    @logger
+    @logger(Field({"foo": Field(str, is_required=False, default_value="bar")}))
     def a_logger(_):
         raise Exception("not executed")
 
     @asset
     def one():
         return 1
 
-    defs = Definitions(assets=[one], loggers={"custom_logger": a_logger})
+    asset_job_with_logger_config = define_asset_job(
+        "custom_logger_config_job",
+        config={"loggers": {"custom_logger": {"config": {"foo": "baz"}}}},
+    )
+
+    @op
+    def my_op():
+        pass
+
+    @job(config={"loggers": {"custom_logger": {"config": {"foo": "baz"}}}})
+    def my_job():
+        my_op()
+
+    # on its own, my_job's config_mapping is invalid because it references a logger
+    # that does not exist
```

Comment:
can we add an assert somewhere that ensures that "baz" is flowed through the config and now "bar"